stephenPrintup/README.md

```diff
@@ -0,0 +1,276 @@
+result      false
+
+**c)** `x == 'x'`
+
+prediction true								<----wrong
+result     undefined
+
+**d)** `x == (x+'')`
+
+prediction undefined
+result     undefined
+
+**e)** `'' == ' '`
+
+prediction  true							<----wrong
+result      false
+
+**f)** `x = true`
+
+prediction  true
+result      true
+
+**g)** `var x; x == 'undefined'`
+
+prediction  undefined							<----wrong
+result      false
+
+**h)** `'9'<'10'`
+
+prediction true
+result     false 				            	<---reveiw
+
+**i)** `typeof x + 1 === "number"`
+
+prediction false, because "number" is a string
+result     false
+
+**j)** `typeof x % 2 === "number"`
+
+prediction false, because "number" is a string
+result     false
+
+**k)** `typeof (x % 2) === "number"`
+
+prediction false, because "number" is a string
+result    true                      				<----review
+
+**l)** `x++ == ++x`
+
+prediction false, x++ will add 1 and ++x will turn x into number
+result     false
+
+**m)** `++x == x++`
+
+prediction false, ++x will produce a number with a left over + and x ++ will add a number
+result     undefined									<---review
+
+**n)** `"1"+x == 1+x`
+
+prediction false, "1" + x produces string and 1 + x could produce string
+result      undefined										<----review
+
+**o)** `"0"+1 == 1`
+
+prediction  false, "0" + 1 == "01"
+result true, "0" + 1 = "01", which is still truthy
+
+**p)** `(typeof (x+1))===(typeof x)`	
+
+prediction false, first is a number second is undefined, ===
+result     undefined											<----wrong
+
+**q)** `(x*1 == x) || ((typeof x) != "number")`
+
+prediction  true, if x is a string it is set to number then a string (left) and on the right it is a string which is not a number
+result      undefined									<----reveiw
+
+**r)** `(x=(typeof (x+(typeof x))))==x`
+
+prediction   true, because nothing is changing
+result       undefined									<----review
+
+---
+
+_All of the following can be solved with ordinary expressions and global variables with primitive values.  You don't need functions, loops, or other topics beyond our first two classes._
+
+ **2)** (_10%_)
+
+Assume variables x, y, and z are numbers.
+
+**a)**
+Write an expression for the mean (i.e. average) of x, y, and z.
+
+(x + y + z)/3
+
+**b)**
+Write a series of expressions to adjust each of x, y, and z halfway toward the mean of the three.
+That is, reset the value of each variable to something new based on its previous value.
+
+var mean = (x + y + z)/3
+x = x/mean
+y = y/mean
+z = z/mean
+
+---
+
+**3)** (_20%_)
+
+Suppose you're encoding geometric shapes in a Cartesian (2D) coordinate system, and you represent a rectangle with four numeric variables:
+
+- _l_ : horizontal position of left edge (relative to some origin);
+- _r_ : horizontal position of right edge;
+- _t_ : vertical position of top edge;
+- _b_ : vertical position of bottom edge.
+
+**a)**
+Write an expression for the rectangle's area.
+
+var width = r - l
+var length = t - b
+var area = width * length
+
```

Comment:
Good job! Keep in mind that semicolons are important. They help separate each declaration and expression. The better syntax would be: "var width = r-l;"